SDWebImage/FLAnimatedImage/FLAnimatedImageView+WebCache.m

```diff
@@ -138,7 +138,13 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                                strongSelf.animatedImage = associatedAnimatedImage;
                                return;
                            }
-                           // Step 2. Check if original compressed image data is "GIF"
+                           // Step 2. Check if image is placeholder image
+                           if (isPlaceholder) {
```

Comment:
We accept the placeholder of `FLAnimatedImage`, this break previous user's usage.